### ISSUE TITLE
Insufficient screenshots in the UG

### ISSUE THREAD

**Issue by Teee728** [Teee728/pe#14]
There is only one screenshot found of the application. There are no screenshots to support the command explanations. Hence, it appears too wordy.

**Comment by nus-se-script** [comment]
# Team's Response
Mostly cosmetic / UI issue but this is a good catch nonetheless.
# Items for the Tester to Verify
## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [x] I disagree

**Reason for disagreement:** I believe that this issue is of ```severity.Medium```. There is barely any evidence of images within the User Guide, hence it would hamper the user's ability to catch certain details.

<catcher-end-of-segment><hr>
